essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-use-case-software-engineer.md

```diff
@@ -1,2 +1,87 @@
 # \[Draft] Use Case: Software Engineer
 
+
+<!-- Introduction -->
+## Introduction
+Welcome, fellow software engineers! Today, I want to share with you how I use Appflowy to stay organized and productive in my work. As software engineers, we all know how important it is to keep track of our projects, deadlines, and tasks. That's where Appflowy comes in! With its simple and intuitive design, Appflowy makes it easy to stay on top of everything.
+
+![image](https://user-images.githubusercontent.com/89600478/235852123-0105d13e-b05e-4df0-b136-4825ca817351.png)
+
+
+
+<!-- How to Contribute -->
+## Installation
+These are the installation instructions for an end user. If you wish to contribute to AppFlowy or if you just want to view and run directly from source code please visit the [Contribute to AppFlowy section](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy).<br>
+You can view the [System Requirements](https://appflowy.gitbook.io/docs/essential-documentation/install-appflowy/requirements).<br>
+You can go directly to the [Installation methods](https://appflowy.gitbook.io/docs/essential-documentation/install-appflowy/installation-methods).
+
+## Setting Up
```

Comment:
After installation, users can pick a folder for saving data on the quick start page.
![image](https://github.com/AppFlowy-IO/AppFlowy-Docs/assets/14248245/8640b0f6-bd4f-4877-b757-51ed3e981f9c)

I think we need to add this part with latest screenshot.

Reply:
Changes has been made for this one, I could not get a screenshot of the quick start page as it only appear once when you freshly install Appflowy, I tried reinstalling too but didn't worked. 
So I put a latest screenshot from the Internet.